packages/lambda/src/functions/serviceApi/versions/v0/routes/ornRoutes.ts

```diff
@@ -26,21 +27,20 @@ export const apiV0LookupOrns = createRoute({name: 'apiV0LookupOrns', method: MET
 );
 
 export const apiV0Search = createRoute({name: 'apiV0Search', method: METHOD.GET, path: '/api/v0/search',
-  requestServiceProvider},
+  requestServiceProvider: composeServiceMiddleware(
+    requestServiceProvider,
+    searchMiddleware,
+  )},
   async(_params: undefined, services) => {
-    const {query: rawQuery, limit: rawLimit, ...rawFilters} = services.request.queryStringParameters || {};
+    const {query: rawQuery, limit: rawLimit} = services.request.queryStringParameters || {};
 
     const query = assertDefined(rawQuery, new InvalidRequestError('an query string is required'));
 
     const limit = rawLimit !== undefined
       ? assertNotNaN(parseInt(rawLimit, 10), new InvalidRequestError('limit must be numeric'))
       : undefined;
 
-    const filters = Object.fromEntries(Object.entries(rawFilters).map(([key, value]) => [key,
-      typeof value === 'string' && value.indexOf(',') > -1 ? value.split(',') : assertString(value, new InvalidRequestError('filters must be strings'))
-    ]));
-
-    const data = await search(query, limit, filters);
+    const data = await search(services.searchClient, query, limit);
```

Comment:
i guess no filters are supported by this api call right? so you just removed that part?

Reply:
Yeah with the changes nothing uses the filters anymore.